### ISSUE TITLE
Blank page

### ISSUE THREAD

**Issue by saitoh183** [d8ahazard/PhlexPublic#1]
Im on ubuntu 16.04 and i am using nginx. I already have other php based sites running in nginx and i added Phlex. So i restarted my nginx, navigated to localhost:8083/Phlex and i get a login page

![image](https://cloud.githubusercontent.com/assets/1534197/23817828/2d19320c-05c4-11e7-8b1c-288599deb150.png)

After i enter my credentials, i just get a blank page. In the log, it seems to have validated my credentials properly.

here is the config.php: (replaced original values)

```
'; <?php die('Access denied'); ?>
[general]
deviceID = "123456789aaaaaaaa"

[user-_-saitoh183]
plexToken = "123456789aaaaaaaa"
plexCred = "123456789aaaaaaaa"
plexUserName = "saitoh183"
```
 and logs:(replaced original values)

```
I/Fri, 10 Mar 2017 18:49:51 -0500: LOGIN.PHP: Posted Username is saitoh183
I/Fri, 10 Mar 2017 18:49:51 -0500: signIn: URL is https://plex.tv/users/sign_in.xml
I/Fri, 10 Mar 2017 18:49:52 -0500: signIn: Result is <?xml version="1.0" encoding="UTF-8"?>
<user email="saitoh183@gmail.com" id="1069973" uuid="123456789aaaaaaaa" mailing_list_status="active" thumb="https://plex.tv/users/123456789aaaaaaaa/avatar?c=2017-03-10+23%3A50%3A03+UTC" username="saitoh183" title="saitoh183" cloudSyncDevice="123456789aaaaaaaa locale="en" authenticationToken="123456789aaaaaaaa" authToken="123456789aaaaaaaa" scrobbleTypes="" pin="123456789aaaaaaaa" restricted="0" home="1" guest="0" queueEmail="queue+cn9zmabMEBpR6z6up3AB@save.plex.tv" queueUid="bbcd095f8d03a5e6" homeSize="15" maxHomeSize="15" rememberMe="false" secure="1" certificateVersion="2">
  <subscription active="1" status="Active" plan="lifetime">
    <feature id="sync"/>
    <feature id="home"/>
    <feature id="camera_upload"/>
    <feature id="content_filter"/>
    <feature id="session_bandwidth_restrictions"/>
    <feature id="session_kick"/>
    <feature id="pass"/>
    <feature id="cloudsync"/>
    <feature id="premium_music_metadata"/>
    <feature id="lyrics"/>
    <feature id="music_videos"/>
    <feature id="trailers"/>
    <feature id="dvr"/>
    <feature id="photo_autotags"/>
    <feature id="hardware_transcoding"/>
    <feature id="webhooks"/>
  </subscription>
  <roles>
    <role id="plexpass"/>
  </roles>
  <entitlements all="1">
    <entitlement id="cpms"/>
    <entitlement id="roku"/>
    <entitlement id="android"/>
    <entitlement id="xbox_one"/>
    <entitlement id="xbox_360"/>
    <entitlement id="windows"/>
    <entitlement id="windows_phone"/>
    <entitlement id="ios"/>
  </entitlements>
  <profile_settings default_audio_language="en" default_subtitle_language="en" auto_select_subtitle="1" auto_select_audio="0"/>
  <providers>
    <provider expires_at="1488963089" id="google_plexcloud"/>
  </providers>
  <username>saitoh183</username>
  <email>saitoh183@gmail.com</email>
  <joined-at type="datetime">2013-06-28 02:17:19 UTC</joined-at>
  <authentication-token>123456789aaaaaaaa</authentication-token>
</user>
```

**Comment by saitoh183** [comment]
Yeah just noticed the CAPS on PHP.5.6...guess i will have to get xampps on windows just for this for now. Will you be supporting 7.X eventually?

**Comment by d8ahazard** [comment]
I'll switch my test environment to 7.x, see why it's not working.

**Comment by d8ahazard** [comment]
Fixed with #2e902e6!
